### DISCUSSION TITLE
Resolvable: false on the @key directive

### DISCUSSION THREAD

**Discussion by PatricioNG** [hayes/pothos#1167] [Q&A] [answered]
Using the federation plugin with Pothos, but a bit unclear on how to set `resolvable` to `false` properly, does anyone have any examples? https://www.apollographql.com/docs/federation/federated-types/federated-directives/#key

**Comment by PatricioNG** [comment, marked as answer]
my bad, found the answer https://github.com/hayes/pothos/issues/1165

**Reply by hayes** [reply]
new version is published now

**Reply by hayes** [reply]
@PatricioNG, not sure if you were following the other thread (are you working on the same project, or did you just happen to run into this at the same time?).

I reverted the previous change and pushed up a new solution, because this got released too quickly because of a misunderstanding on my end.

**Reply by PatricioNG** [reply]
Ah yeah @hayes I am working on the same project just didn't realize Cameron already opened a note 😅 Thanks for that though, I saw the updates and they look good! Will let Cameron know 😄
